### PR TITLE
TRUNK-5454: Refactor WebUtil class not to be a GlobalPropertyListener

### PR DESCRIPTION
**Pull request by samuelmale**
Ticket : https://issues.openmrs.org/browse/TRUNK-5454

### CODE REVIEW

**Comment by samuelmale** [comment]
Just out of curiosity; why was this closed @dkayiwa ?

**Comment by dkayiwa** [comment]
Sorry i should have commented while closing. The work done is way much more than the ticket asked for. But i can reopen it if you want to reuse.

**Comment by samuelmale** [comment]
Did you get some time to look at the ticket link of this PR @dkayiwa ?

**Comment by samuelmale** [comment]
@dkayiwa  ping

**Comment by dkayiwa** [comment]
On the ticket, can you give a link to the sonar report?

**Comment by samuelmale** [comment]
I added a comment for you on the ticket

File: web/src/main/java/org/openmrs/web/WebUtil.java [hunk 1]
```diff
@@ -412,31 +408,4 @@ public static String formatDate(Date date, Locale locale, FORMAT_TYPE type) {
 		return date == null ? "" : dateFormat.format(date);
 	}
 	
-	/**
-	 * @see org.openmrs.api.GlobalPropertyListener#supportsPropertyName(java.lang.String)
-	 */
-	@Override
-	public boolean supportsPropertyName(String propertyName) {
-		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
```
Comment (dkayiwa) [inline web/src/main/java/org/openmrs/web/WebUtil.java]: This method has nothing to do with defaultDateCache. So removing the implemented interface may have unintended consequences.

**Comment by dkayiwa** [comment]
I think the surgical way of dealing with this in a safe manner is to simply removed the variable and methods dealing with it, without necessarily removing the interface. Interface could eventually be removed but this requires careful thought and investigation

**Pull request closed without merging**